Taskfile.yml

```diff
@@ -132,7 +133,13 @@ tasks:
     desc: "Fill `test` database with data for experiments"
     cmds:
       - bin/task{{exeExt}} -d integration env-data
-
+  
+  env-repl:
+    desc: "A minimal environment for testing a single-node replica set"
+    cmds:
+      - >
+        docker compose up --always-recreate-deps --force-recreate --remove-orphans --renew-anon-volumes --timeout=0 --detach --build --pull=always {{.REPLSET}}
```

Comment:
Is always pulling new image expected? Let's mention that in the description too :hugs:

Reply:
It's not required to always pull a new image. I removed unnecessary flags.